addons/udes_stock/tests/test_stock_quant_package.py

```diff
@@ -0,0 +1,309 @@
+# -*- coding: utf-8 -*-
+from .common import BaseUDES
+from odoo.exceptions import ValidationError
+from collections import defaultdict
+
+
+class TestStockQuantPackageModel(BaseUDES):
+    @classmethod
+    def setUpClass(cls):
+        super(TestStockQuantPackageModel, cls).setUpClass()
+        cls.Package = cls.env['stock.quant.package']
+        cls.test_package = cls.Package.get_or_create('test_package_01', create=True)
+        cls.create_quant(
+            cls.apple.id,
+            cls.test_stock_location_01.id,
+            10,
+            reserved_quantity=5,
```

Comment:
Why reserved 5?

Reply:
Used in checking the amount of reserved quantity in a package later

Reply:
This can be changed in the test later.